treeherder/perf/tasks.py

```diff
@@ -1,11 +1,11 @@
 import newrelic.agent
-from celery import task
 
 from treeherder.perf.alerts import generate_new_alerts_in_series
 from treeherder.perf.models import PerformanceSignature
+from treeherder.workers.task import retryable_task
 
 
-@task(name='generate-alerts')
+@retryable_task(name='generate-alerts', max_retries=10)
```

Comment:
@wlach - just checking retrying `generate_alerts()` is fine?

Reply:
should be, it's designed to not cause any problems if called repeatedly